scripts/create_ts_for_integration_tests.py

```diff
@@ -84,7 +90,7 @@ def create_edge_case_time_series(ts_api):
     print(f"Created {len(ts_lst)} sparse ts with data")
 
 
-def create_dense_time_series(ts_api):
+def create_dense_time_series() -> Tuple[List[TimeSeries], List[pd.DataFrame]]:
```

Comment:
Why do you (randomly) add type hints here and not for any of the other functions? 👀 🤔

Reply:
Inconsistent force of habit :)